modules/ROOT/pages/access-developer-portal-task.adoc

```diff
@@ -1,25 +1,24 @@
-= Accessing the Developer Portal
+= Access the Developer Portal
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-API portals listed on an organization's developer portal that are public, or to which you have been granted Portals Viewer access, are visible to you. From the public Anypoint Platform Developer portal, you can access Anypoint Platform APIs to automate Anypoint Platform activities.
+As an administrator, you can view API portals if you have the appropriate access. The public https://anypoint.mulesoft.com/apiplatform/anypoint-platform/#/portals[Anypoint Platform developer portal], enables you to automate Anypoint Platform activities.
+ 
+You can view an API portal in Anypoint Platform if:
 
-In API Manager, from the API Administration page, use the hamburger menu on the right. Alternatively, use the following procedure.
+* The portal is listed in the organization's public developer portal
+* You are granted Portals Viewer access for a portal 
 
-To access the Anypoint Platform Developer Portal:
 
-The Anypoint Platform developer portal is a public portal that you can find by searching on the internet or by clicking the link at the bottom of this page.
+To access your organization's Anypoint Platform developer portal:
```

Comment:
This step is to go the the portal of one asset. So instead of `To access your organization's` would be `To access an API Instance portal`. *API Manager* > *Administration*, etc...

To access the public portal of an organization:

Go To Exchange > Public Portal (link on the sidebar)

Reply:
Done! Thanks!